README.md

```diff
@@ -78,6 +85,16 @@ List of available resize filters:
 - `mitchell` => Resize using Mitchell-Netravali filter
 - `lanczos3` => Resize using Sinc-windowed Sinc with radius of 3
 
+## Example
```

Comment:
If not hard, please add examples for image resizing and quantization

Reply:
ok, I'll add another example for that.